options.nix

```diff
@@ -191,7 +188,7 @@ in
     commands = [
       {
         help = "prints this menu";
-        name = "devshell-menu";
+        name = "${config.name}-menu";
```

Comment:
I'm not sure about that

Reply:
My reasoning was:

_Show me the help for that particular **instance** of devshell which is identified by the name of `${config.name}`_

On the contrary, it looses devshell branding.

Ultimately, this is akin to a marketing decision.

Reply:
It could also be renamed to `menu` so it's shorter to type.

Reply:
I agree and will adapt accordingly.